argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enable support for namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+The {gitops-title} enables support for two modes of Argo Rollouts installation:
+
+* *Cluster-scoped installation* (default): The Argo Rollouts custom resources (CRs) defined in any namespace are reconciled by the Argo Rollouts instance. This allows you to use Argo Rollouts CR across any namespace on the cluster.
+* *Namespace-scoped installation*: The Argo Rollouts instance is installed in a specific namespace and only handles Argo Rollouts CR within the same namespace. This installation mode includes the following benefits:
+** This mode does not require cluster-wide `ClusterRole` or `ClusterRoleBinding` permissions. Users may install and use Argo Rollouts within a single namespace without requiring cluster permissions.
+** This mode provides security benefits by limiting the cluster scope of a single Argo Rollouts instance to a specific namespace.
+
+[NOTE]
+====
+{gitops-title} does not allow both modes of Argo Rollouts installation at the same time to prevent unintended privilege escalation.
+====
+
+Perform the following steps to switch between cluster-scoped and namespace-scoped Argo Rollouts installations.
```

Comment:
```suggestion
To switch between cluster-scoped and namespace-scoped Argo Rollouts installations, complete the following steps:
```

Reply:
IBM Style Word Usage advises to avoid the term _perform_ in technical content where possible.

Reply:
Fixed.